src/components/filters/filter-dynamic/filter-dynamic.component.html

```diff
@@ -21,7 +21,7 @@
         <hr>
 
         <li *ngIf="showTypeahead" class="typeahead-box">
-            <input [(ngModel)]="searchQuery" [typeahead]="typeaheadItems" class="form-control" 
+            <input #inputBox [(ngModel)]="searchQuery" [typeahead]="typeaheadItems" class="form-control" 
```

Comment:
I don't think we need to do all this element ref stuff. We should just be able to do `this.searchQuery = ""` in the code anywhere we want to clear it?